independent-publisher-connectors/Companies House/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Authorization",
+          "description": "The Authorization for this api",
+          "tooltip": "Provide your Authorization",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#149CFA",

```

Comment:
@MattCollins-Jones Please go thru the [template provided for apiProperties.json](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json) file, all the IP Connectors must have iconBrandColor as "#da3b01"